src/components/Home/Home.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Slider from '../../components/Slider/Slider';
+import ArticlesCategories from '../../components/Categories/Categories';
+import Modal from '../../components/Modal/Modal';
+import Header from '../../components/Header';
+import Feedback from '../../components/Feedback/Feedback';
+import Link from 'gatsby-link'
+import AvocadoTestImage from '../../pages/images/avocado.png';
+import {TEST_ARTICLES} from "../../pages/index";
+
+export default class HomeComponent extends React.Component {
+	constructor(props) {
+		super(props);
+		TEST_ARTICLES.map(article => {
+			article.more = false;
+		});
+		this.state = {
+			articles: props.articles || [],
+			modalConfig: {
+				id: "#about",
+				title: "About Service",
+				additionals: "Content here"
+			}
+		}
+	}
+	componentDidMount() {
+		this.state.articles ? this.state.articles.map(news => {
+			news.more = false;
+		}) : false;
+	}
+	showLittleMore(param) {
+		const index = this.state.articles.map(function(e) { return e.title }).indexOf(param.title);
```

Comment:
Please, use the ES6 destructuring and arrow functions
```
const { articles } = this.state;
```
+ you can try to use the filter method
you can omit the else statement

Reply:
i think all this is not important